app/assets/stylesheets/active_admin/components/_date_picker.scss

```diff
@@ -1,147 +1,119 @@
-        height: 0;
-        margin: 0px 4px 0px 0px;
-        border-top: 5px solid transparent;
-        border-left: 5px solid white;
-        border-bottom: 5px solid transparent;
-      }
+  // base for arrows
+  &:after {
+    border-bottom: 4px solid transparent;
+    border-top: 4px solid transparent;
+    content: '';
+    height: 0;
+    position: absolute;
+    top: calc(50% - 4px);
+    width: 0;
+  }
 
-      span {
-        display: none;
-      }
-    }
+  span {
+    display: none; // hide Next/Prev text
+  }
+}
+
+.ui-datepicker-prev {
+  left: 0;
+
+  &:after {
+    border-right: 5px solid; // arrow pointing left
+    left: calc(50% - 2px); // half of arrow size (border) to horizontally center
   }
+}
 
-  table.ui-datepicker-calendar {
-    @include rounded-bottom;
-    @include shadow(0,1px,6px,rgba(0,0,0,0.26));
-    background-color: #f4f4f4;
-    border: solid 1px #63686e;
-    left: 2px;
-    margin-bottom: 0px;
-    position: relative;
-    top: -2px;
-    width: 156px;
-
-    td, th {
-      padding: 0px;
-      text-align: center;
-    }
+.ui-datepicker-next {
+  right: 0;
 
-    thead th {
-      background-color: #dbdddf;
-      color: #333333;
-      font-weight: normal;
-      font-size: 0.8em;
-      padding-top: 1px;
-    }
+  &:after {
+    border-left: 5px solid; // arrow pointing right
+    right: calc(50% - 2px); // half of arrow size (border) to horizontally center
+  }
+}
+
+.ui-datepicker-calendar {
+  background-color: $date-picker-calendar-background-color;
+  border: none;
+  margin-bottom: 0;
+
+  td, th {
+    padding: 0;
+    text-align: center;
+  }
+
+  thead th {
+    border-bottom: none;
+    font-size: 0.9em;
+    font-weight: $font-weight-bold;
+    padding: 2px 0;
+  }
+
+  tbody {
+    td {
+      border: none;
+      height: 22px;
+      width: 22px;
+
+      a {
+        border-radius: $border-radius-base;
+        color: $text-color;
+        display: block;
+        font-size: 0.9em;
+        line-height: 1;
+        padding: 6px 0;
+
+        &:hover {
+          background-color: $date-picker-calendar-link-hover-background-color;
+        }
+
+        &.ui-state-active,
+        &.ui-state-active:hover {
+          background-color: $date-picker-calendar-link-active-background-color;
+        }
 
-    tbody {
-      color: #666666;
-
-      td {
-        border: none;
-        height: 24px;
-        width: 22px;
-
-        a {
-          @include rounded;
-          color: #666666;
-          font-weight: bold;
-          font-size: 0.85em;
-          padding: 4px;
-
-          &.ui-state-active {
-            background-color: #5a5f64;
-            color: #fff;
-            &.ui-state-hover {
-              background-color: #5a5f64;
-              color: #fff;
-            }
-          }
-          &.ui-state-hover {
-            background-color: #eceef0;
-          }
-          &.ui-state-highlight {
-            background-color: #dbdddf;
-          }
+        &.ui-state-highlight,
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4